### ISSUE TITLE
Super build compilation issues

### ISSUE THREAD

**Issue by Shoegzer** [libretro/libretro-super#500]
I've discovered the following cores require manual compiling, since they don't build with the super script: **fsuae, mednafen_saturn, melonds, pcem, parallel_n64, redream, reicast**

Also, the **dolphin** and **pokemini** cores are not included with super script at all.

Can someone please update the super script to fix these?  Thanks.

**Comment by orbea** [comment]
What OS and compiler are you experiencing these build failures? Can you paste the errors?

**Comment by Shoegzer** [comment]
They aren't build failures, but libretro-build.sh doesn't seem to compile them so they have to be compiled manually.

**Comment by orbea** [comment]
What OS? What exactly doesn't work? Can you paste any errors?

I tested this and was able to build the reicast core.
```
./libretro-fetch.sh reicast
./libretro-build.sh reicast
```

**Comment by Shoegzer** [comment]
The system I tested with was Linux Mint 18.2 but it shouldn't matter.  My point is that if you try libretro-built.sh by itself, it should build all the systems in the super build.  Currently, it builds MOST of them, excepting those I've listed above.

**Comment by orbea** [comment]
Okay, I see. They are just not included in the default choices....

**Comment by Shoegzer** [comment]
Yup, I think you are right.  My guess is that libretro-fetch needs an update for dolphin and pokemini, and libretro-build needs an update for dolphin, pokemini, reicast and the others i've mentioned above.

**Comment by retro-wertz** [comment]
up for these. i want to request to add the following cores to libretro-build.sh for

libretro-build-core px68k
libretro-build-core mednafen_saturn
libretro-build-core atari800

im not familiar enough with other platforms to categorize them so hope somebody will update the script.

Thanks.

**Comment by Shoegzer** [comment]
Updated this issue since most cores have compiled just fine for some time now.

@orbea would you consider adding the pcem core to the buildbot scripts so this can be closed out?  My thoughts, realizing it hasn't had an update in a few years, it's still somewhat useful as an RA core and may as well be listed in the script for anyone who wants to try it.  Including it with the buildbot script would also raise the profile so others are more likely to contribute updates.  What do you think?

**Comment by MiltosKoutsokeras** [comment]
Currently (commit b41b86859d9deadd3794b7d4345f5dc4e8697195) the dolphin core does not build, as it has no Makefile at all and the rule is the following:

```
include_core_dolphin() {
	register_module core "dolphin"
}
libretro_dolphin_name="Dolphin"
libretro_dolphin_git_url="https://github.com/libretro/dolphin.git"
libretro_dolphin_build_subdir="Source/Core/DolphinLibretro"
libretro_dolphin_build_makefile="Makefile"
```

This is completely wrong, the core needs CMake and the build sub directory does not contain a make file either. Is this repository outdated?
